visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.ts

```diff
@@ -181,33 +181,35 @@ export class CodeMapMouseEventService
 
 			this.threeCameraService.camera.updateMatrixWorld(false)
 
-			if (mapMesh) {
-				this.raycaster.setFromCamera(mouseCoordinates, camera)
+			if (mapMesh && camera) {
+				if (camera.isPerspectiveCamera) {
+					this.raycaster.setFromCamera(mouseCoordinates, camera)
```

Comment:
I'm in the process of refactoring the labels, I would commit this as it is now to avoid failing tests and fix it during refactoring. Is this fine by you?

Reply:
Yes, that's fine :-) It was just a suggestion.